nipype/interfaces/petpvc.py

```diff
@@ -0,0 +1,235 @@
+Multi-target correction - "MTC"
+MTC with Labbe - "LABBE+MTC"
+MTC with Van-Cittert - "MTC+VC"
+MTC with Richardson-Lucy - "MTC+RL"
+MTC with Labbe and Van-Cittert - "LABBE+MTC+VC"
+MTC with Labbe and Richardson-Lucy- "LABBE+MTC+RL"
+Iterative Yang - "IY"
+Iterative Yang with Van-Cittert - "IY+VC"
+Iterative Yang with Richardson-Lucy - "IY+RL"
+Muller Gartner - "MG"
+Muller Gartner with Van-Cittert - "MG+VC"
+Muller Gartner with Richardson-Lucy - "MG+RL"
+
+"""
+from __future__ import print_function
+from __future__ import division
+
+import os
+import warnings
+
+from nipype.interfaces.base import (
+    TraitedSpec,
+    CommandLineInputSpec,
+    CommandLine,
+    File,
+    isdefined,
+    traits,
+)
+
+warn = warnings.warn
+
+pvc_methods = ['GTM',
+               'IY',
+               'IY+RL',
+               'IY+VC',
+               'LABBE',
+               'LABBE+MTC',
+               'LABBE+MTC+RL',
+               'LABBE+MTC+VC',
+               'LABBE+RBV',
+               'LABBE+RBV+RL',
+               'LABBE+RBV+VC',
+               'MG',
+               'MG+RL',
+               'MG+VC',
+               'MTC',
+               'MTC+RL',
+               'MTC+VC',
+               'RBV',
+               'RBV+RL',
+               'RBV+VC',
+               'RL',
+               'VC']
+
+
+class PETPVCInputSpec(CommandLineInputSpec):
+    in_file   = File(desc="PET image file", exists=True, mandatory=True, argstr="-i %s")
+    out_file  = File(desc="Output file", genfile=True, hash_files=False, argstr="-o %s")
+    mask_file = File(desc="Mask image file", exists=True, mandatory=True, argstr="-m %s")
+    pvc       = traits.Enum(pvc_methods, desc="Desired PVC method", mandatory=True, argstr="-p %s")
+    fwhm_x    = traits.Float(desc="The full-width at half maximum in mm along x-axis", mandatory=True, argstr="-x %.4f")
+    fwhm_y    = traits.Float(desc="The full-width at half maximum in mm along y-axis", mandatory=True, argstr="-y %.4f")
+    fwhm_z    = traits.Float(desc="The full-width at half maximum in mm along z-axis", mandatory=True, argstr="-z %.4f")
+    debug     = traits.Bool (desc="Prints debug information", usedefault=True, default_value=False, argstr="-d")
+    n_iter    = traits.Int  (desc="Number of iterations", default_value=10, argstr="-n %d")
+    n_deconv  = traits.Int  (desc="Number of deconvolution iterations", default_value=10, argstr="-k %d")
+    alpha     = traits.Float(desc="Alpha value", default_value=1.5, argstr="-a %.4f")
+    stop_crit = traits.Float(desc="Stopping criterion", default_value=0.01, argstr="-a %.4f")
+
+
+class PETPVCOutputSpec(TraitedSpec):
+    out_file = File(desc = "Output file")
+
+
+class PETPVC(CommandLine):
+    """ Use PETPVC for partial volume correction of PET images.
+
+    Examples
+    --------
+    >>> from ..testing import example_data
+    >>> #TODO get data for PETPVC
+    >>> pvc = PETPVC()
+    >>> pvc.inputs.in_file   = example_data('pet.nii.gz')
```

Comment:
after adding the docstring in the beginning of the file as I described in my other comment you can drop the `example_data` and just do `pet.nii.gz`